Dimodi.Telerik.Blazor.Templates.csproj

```diff
@@ -16,18 +16,19 @@
     <PackageReadmeFile>README.md</PackageReadmeFile>
     <PackageReleaseNotes>
 NEW
-* Moved TelerikRootComponent to a separate layout file TelerikLayout.razor.
-* Added a project template option to use Telerik Document Processing NuGet packages for working with CSV, DOCX, HTML, PDF and XLXS files.
-* Removed the obsolete Web App (Server) and Web App (WebAssembly / Auto) project templates. Use the unified Web App template instead.
-* Removed the obsolete .NET 6 and .NET 7 project templates.
+* Added support for the latest .NET 10 Preview. Always use preview .NET versions with the latest Telerik UI for Blazor version.
+
+FIXED
+* Do not load package Microsoft.Extensions.Localization in the server project of Blazor Web Apps with WebAssembly or Auto render mode.
+* Do not import the Microsoft.Extensions.Localization namespace in Program.cs, as IStringLocalizer is not used.
```

Comment:
```suggestion
* Do not import the Microsoft.Extensions.Localization namespace in Program.cs, as IStringLocalizer is not used.
* Target .NET 8 by default, instead of .NET 9, until .NET 10 becomes official.
```